.ci/docker/common/install_inductor_benchmark_deps.sh

```diff
@@ -48,4 +48,4 @@ install_huggingface
 install_timm
 
 # Clean up
-conda_run pip uninstall -y torch torchvision torchaudio triton
+conda_run pip uninstall -y torch torchvision torchaudio triton torchao
```

Comment:
I just realize that there is one more permission error bug in this script.  After installing TorchBench on line 37, the script runs under root as part of the Docker build, so another `chown -R jenkins /opt/conda` is needed similar to https://github.com/pytorch/pytorch/blob/main/.ci/docker/common/install_conda.sh#L21.  This will fix the remaining error in this script